maintainers/maintainer-list.nix

```diff
@@ -9740,6 +9740,16 @@
     githubId = 5047140;
     name = "Victor Collod";
   };
+  munksgaard = {
```

Comment:
Please use a different commit for this bit, with the commit message: `maintainers: add munksgaard`